### PR TITLE
PD-651 Include 'page' event

### PR DESCRIPTION
**Pull request by edwardupton**

### CODE REVIEW

File: src/common/constants.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const LittledataScriptVersion = '11.1.2';
```
Comment (lyykfi) [inline src/common/constants.ts:1]: Let migrate tracker to release script on this Thus together :-)
less manual work

**Review by lyykfi — CHANGES_REQUESTED** [review]

File: src/common/constants.ts (hunk 1 above)
Comment (edwardupton) [reply src/common/constants.ts:1]: OK!

File: src/segmentTracker/helpers/addEmailToEvents.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { CustomWindow } from '../../..';
 declare let window: CustomWindow;
 
 export const addEmailToTrackEvents = ({ payload, next }: MiddlewareOptions): void => {
-	if (payload.action() !== 'track') return next(payload);
+	if (!['track', 'page'].includes(payload.action())) return next(payload);
```
[the remark this answers is not in the document]
Comment (edwardupton) [reply src/segmentTracker/helpers/addEmailToEvents.ts]: @lyykfi done

File: src/segmentTracker/constants.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+export const segmentEventTypes = {
```
Comment (lyykfi) [inline src/segmentTracker/constants.ts:1]: ❤️

**Review by lyykfi — APPROVED** [review]

**Pull request merged**